icicle/include/icicle/common.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <cstdint>
+
+namespace icicle {
+
+  // Define limb type for hashing
+  using limb_t = uint32_t;
```

Comment:
this should be taken rom Field.

Reply:
it's not defined anywhere currently.
I could define it in field.h  instead but I prefer to avoid it since I don't want to include it everywhere if I don't need it.
I think that field.h should include this file since limb is not restricted to field elements but a more general concept.